.gitignore

```diff
@@ -0,0 +1,45 @@
+### SublimeText ###
```

Comment:
Add README with the proper documentation.

Reply:
readme added